spark/dl/src/test/scala/com/intel/analytics/bigdl/utils/LoggerFilterSpec.scala

```diff
@@ -56,6 +56,7 @@ class LoggerFilterSpec extends FlatSpec with BeforeAndAfter with Matchers {
   }
 
   "A LoggerFilter" should "output correct info on console and bigdl.log" in {
+    TestUtils.cancelOnWindows()
```

Comment:
In this test suite, a lot of linux local folder path is used. Should we change to temp foler? @i8run